kobuki_node/src/diagnostics.cpp

```diff
@@ -26,143 +28,167 @@ namespace kobuki_node
 ** Implementation
 *****************************************************************************/
 
-void BatteryTask::run(diagnostic_updater::DiagnosticStatusWrapper &stat) {
-  switch ( status.level() ) {
-    case ( kobuki::Battery::Maximum ) : {
+void BatteryTask::run(diagnostic_updater::DiagnosticStatusWrapper &stat)
+{
+  switch (status_.level()) {
+    case kobuki::Battery::Maximum:
       stat.summary(diagnostic_msgs::msg::DiagnosticStatus::OK, "Maximum");
       break;
-    }
-    case ( kobuki::Battery::Healthy ) : {
+    case kobuki::Battery::Healthy:
       stat.summary(diagnostic_msgs::msg::DiagnosticStatus::OK, "Healthy");
       break;
-    }
-    case ( kobuki::Battery::Low ) : {
+    case kobuki::Battery::Low:
       stat.summary(diagnostic_msgs::msg::DiagnosticStatus::WARN, "Low");
       break;
-    }
-    case ( kobuki::Battery::Dangerous ) : {
+    case kobuki::Battery::Dangerous:
```

Comment:
Never liked this much. Technically correct, but cognitively expensive given that it looks like separate code blocks and the emphasis to put curly scope around every other code block. Always seems to lead to raised eyebrows and problems for people the first time they hit them.